integration-tests/e2e/kubetest/description/1.20/working.json

```diff
@@ -475,5 +475,28 @@
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support non-existent path", "groups": ["fast"]},
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support readOnly directory specified in the volumeMount", "groups": ["fast"]},
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support restarting containers using directory as subpath [Slow]", "groups": ["fast"]},
-  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] volumes should allow exec of files on the volume", "groups": ["fast"]}
+  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] volumes should allow exec of files on the volume", "groups": ["fast"]},
+  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Inline-volume (default fs)] subPath should verify container cannot write to subpath readonly volumes [Slow]"},
```

Comment:
there are lot of testcases without a group assigned. 
@dguendisch do you know if we have a test definition for such cases or do they never get executed?